custom_components/myskoda/config_flow.py

```diff
@@ -43,9 +44,13 @@ async def validate_options_input(
 
     if CONF_POLL_INTERVAL in user_input:
         polling_interval: int = user_input[CONF_POLL_INTERVAL]
-        if CONF_POLL_INTERVAL_MIN <= polling_interval <= CONF_POLL_INTERVAL_MAX:
-            return user_input
-        raise SchemaFlowError("invalid_polling_interval")
+        if not CONF_POLL_INTERVAL_MIN <= polling_interval <= CONF_POLL_INTERVAL_MAX:
+            raise SchemaFlowError("invalid_polling_interval")
+
+    if CONF_SPIN in user_input:
+        s_pin: str = user_input[CONF_SPIN]
+        if not s_pin.isdigit():
+            raise SchemaFlowError("invalid_spin_format")
```

Comment:
Do we want to validate the `s-pin` here with the `MySkoda.verify_spin()` from https://github.com/skodaconnect/myskoda/pull/175/files ?

Reply:
I think prefer to use `MySkoda.verify_spin()` when storing the S-PIN in config, instead of every time when calling a feature that requires the S-PIN